docs/src/plugin_deployment/plugin_deployment_using_chart_in_bundle.md

```
@@ -21,7 +21,7 @@ helm install -n lb-csi-ns --create-namespace lb-csi helm/lb-csi/
 helm list --namespace=kube-system
 
 NAME  	NAMESPACE  	REVISION  UPDATED        	STATUS  	CHART              	 APP VERSION
-lb-csi	kube-system	1         2024-12-04... 	deployed	lb-csi-plugin-0.13.0	 1.17.0
+lb-csi	kube-system	1         2025-03-11... 	deployed	lb-csi-plugin-1.18.0	 1.17.0
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 7

(MD010, no-hard-tabs)